src/pytest/test_pair.py

```diff
@@ -15,7 +15,17 @@
     ['potential', 'params', 'r_cut', 'shift', 'distance', 'energy', 'force'],
 )
 
-potential_tests = []
+potential_tests = [
+    PotentialTestCase(
```

Comment:
Add test cases for your potential here that you have computed by hand, with Mathematica or Python, etc. The forces and energies need to be recorded to 4 decimal places. See the named tuple above for an explanation of the meaning of each of these. You can (should) add multiple test cases for each of your potentials.

Reply:
Also, add a short comment explaining the reasoning for each test, i.e "testing that potential and force are zero outside of cutoff" or "testing that if epsilon is zero, the potential is zero" etc.